environments/editors.md

```diff
@@ -28,34 +28,28 @@ etc.
 
 ![VS Code Remote Explorer](../assets/vscode-remote-ssh-panel.png)
 
-## VS Code in the browser
+## VS Code in the Browser
 
-Launch VS Code in the browser from the Environments page by clicking the _Code_ icon.
+Launch VS Code in the browser from the Environments page by clicking the _Code
+Web_ icon.
 
 ![Launch an Environment](../assets/launch-env.png)
 
-## JetBrains in the browser
+## JetBrains IDEs in the Browser
 
-Click your avatar to open a menu. Select **Feature Preview** to open the feature
-preview modal where you can enable JetBrains support.
-
-![Enable JetBrains Support](../assets/enable-jetbrains-support.png)
-
-Coder launches remote IDEs in their own windows; be sure to set your
-browser to allow popup windows so that you can use your IDE.
-
-With support enabled and the editor installed with the Image, you'll be able to
-launch the editor from the dashboard.
+If your image includes a JetBrains IDE, you can launch it from the dashboard.
+Coder launches JetBrains IDEs in their own windows; be sure to set your browser
+to allow popup windows so that you can use your IDE.
 
 ![JetBrains Logos](../assets/jetbrains-launcher-icons.png)
 
 > If you need a valid license to run your IDE locally, you'll also need one to
 > run it in Coder.
 
-### Install JetBrains' IDEs
+### Manually Installing JetBrains' IDEs
 
-You can manually install JetBrains' IDEs. After following JetBrains' steps for
-installing your IDE, create a symlink, and add it to $PATH.
+You can also manually install JetBrains' IDEs. After following JetBrains' steps
+for installing your IDE, create a symlink, and add it to $PATH.
```

Comment:
Will this work? JetBrains' recommended install instructions involve installing it into the `/opt` directory, which is destroyed every time a workspace is rebuilt. Perhaps add a note that JetBrains has to be installed in the user's home directory if it isn't being added into the image?

https://www.jetbrains.com/help/idea/installation-guide.html#toolbox

Reply:
I guess it'd work if you did it in your personalize script? But yeah the home directory seems like a better place